source/jormungandr/jormungandr/instance.py

```diff
@@ -129,10 +129,17 @@ def parse_and_get_olympics_forbidden_uris(dict_olympics_forbidden_uris):
         if p not in dict_olympics_forbidden_uris:
             logging.getLogger(__name__).error('olympic_criteria: invalid parameter, {} not found'.format(p))
             return None
+    if not isinstance(dict_olympics_forbidden_uris.get('max_duration_wheelchair'), int):
+        logging.getLogger(__name__).error(
+            'olympic_criteria: invalid parameter, max_duration_wheelchair not found'
```

Comment:
The error message is a bit incorrect? It might have been found but is not a `int`?

```suggestion
            'olympic_criteria: invalid parameter, max_duration_wheelchair is not an integer'
```

Reply:
:+1:

Reply:
Done